Doc 1281/changelog 18 sept (#420)
swan-io/docs: changelog/2025/2025-09-18.mdx

---
title: 18 September 2025
type: release
---

## Improvements

🛒 Improvements to [merchant profiles and merchant payment methods](/topics/merchants/), and the `requestMerchantProfile` and `requestMerchantProfileUpdate` mutations:

- The Testing APIs `simulateMerchantProfileRequestOutcome` and `simulateMerchantPaymentMethodRequestOutcome` now support the new `WaitingForInformation` status and return a list of `verificationRequirements`.

> - A new optional field `expectedMerchantProfileAdditionalInformation` is now available at the `projectInfo` level. This field is only available upon prior agreement with Swan and allows you to customize information collected for merchant profiles in specific use cases. 

Reviewer: If it's a new field, it doesn't hurt to remind partners it has been added to our API architecture imho. I don't see any mention anywhere else in our docs, changelog is the minimum we should do :)

  The information can then be provided using the `additionalInformation` field in merchant profiles and in the `requestMerchantProfile` and `requestMerchantProfileUpdate` mutations.
- The `invoiceExample` field has been deprecated.

> - The `expectedMonthlyPaymentVolume`, `merchantWebsite`, `socialNetwork`, `supportWebsite`, and `termsAndConditions` fields are being deprecated and replaced with the new fields: `expectedMonthlyMerchantProcessingVolume`, `merchantWebsiteUrl`, `socialNetworkUrl`, `supportWebsiteUrl`, and `termsAndConditionsUrl`.

Reviewer 2: We usually write it like this (from previous changelogs) to keep them together and limit repetition: 
On February 7th, we're removing the addMerchantProfile mutation to keep our API consistent. Instead, call the [requestMerchantProfile mutation](https://docs.swan.io/topics/merchants/profiles/guide-request). We're also deprecating the addMerchantProfile mutation today.

> - We've deprecated and replaced `rejectReasons` with `rejectionReasons` for rejected merchant profiles and merchant payment methods.

Author: Same reasoning as above. 

About the `rejectReasons` deprecation - yes we mention it twice but not the same context. We actually say that they'll "will **become mandatory** in the Testing APIs `simulateMerchantProfileRequestOutcome` and `simulateMerchantPaymentMethodRequestOutcome`"

## API updates

### Upcoming breaking changes

📡 Starting **October 9th**, bulk SEPA Credit Transfers from individual account holders will be rejected if [verification tokens](/preview/vop/guide-verify-beneficiary/#step-3) aren't included in the payment initiation. You must either implement the `verifyBeneficiary` mutation (recommended) or restrict bulk credit transfer access to company accounts only. This change applies to both Live and Sandbox environments.

> 💳 On **October 16th**, the `CardUrl` field will be removed. This field is being deprecated. Use `cardMaskedNumber`, `expiryDate`, and `cardDesignUrl` instead.

Reviewer 2: Do we need to say both removed and deprecated? Recommend just using one.

Reviewer: Deprecated for now - removed later

🛒 On **October 16th**, the following merchant onboarding and payment methods breaking changes will take effect:

- A new `WaitingForInformation` status and `verificationRequirements` list will be available for merchant profiles and merchant payment methods to communicate missing information. These requirements must be provided at the merchant profile level using the `requestMerchantProfile` and `requestMerchantProfileUpdate` mutations. 
- A new `supportingDocumentCollections` field will also be available at the merchant profile level for providing required documents.

> - The fields `invoiceExample`, `expectedMonthlyPaymentVolume`, `merchantWebsite`, `socialNetwork`, `supportWebsite`, `termsAndConditions` will be **removed** from the API.

Reviewer 2: I'm guessing it starts being deprecated now, and finally removed on this date? Is that correct?

> - The fields `invoiceExample`, `expectedMonthlyPaymentVolume`, `merchantWebsite`, `socialNetwork`, `supportWebsite`, `termsAndConditions` will be **removed** from the API.

Reviewer 2: Suggest removing "from the API" - since this is all above breaking changes in the API

> - The fields `invoiceExample`, `expectedMonthlyPaymentVolume`, `merchantWebsite`, `socialNetwork`, `supportWebsite`, `termsAndConditions` will be **removed** from the API.

Reviewer: Yes, that is standard practice: deprecate to allow for integration time, then remove to force migration :)

- `rejectReasons` will be **removed** for card payment methods.
- `rejectionReasons` for `Rejected` merchant profiles and merchant payment methods will become **mandatory** in the Testing APIs `simulateMerchantProfileRequestOutcome` and `simulateMerchantPaymentMethodRequestOutcome`.
- Dynamic rolling reserve changes and maximum payment amount limits will be introduced at the merchant payment method level to mitigate risk. Subscribe to the `MerchantPaymentMethod.Created` and `MerchantPaymentMethod.Updated` webhooks for these updates.

💸 Effective **October 19th**, we're introducing updates to the `initiateCreditTransfers` endpoint to improve its usability and clarity:

- The `save` parameter will now default to `false`.
- The `mode` parameter is now a **required** field. The default value is set to `Regular`.
- The `isMyOwnIban` and `beneficiaryId` input fields have been **removed**.
- The API will now return a `ValidationRejection` error instead of `ForbiddenRejection` when an input contains **invalid** data.

## Update

> 🧭 Starting **October 21st**, card-issuing spending limits will be [aggregated at the hour level](/topics/cards/#hour-level-aggregation). 

Author: @max-koro , unsure why this link doesn't show up in the preview despite being live in the docs. Do you have an idea?

Reviewer: Did you rebase?

> For spending-limit computation, each payment will be timestamped at the first second of the first minute of the hour, and each hour will reset the rolling period.

Reviewer 2: This needs to be more plain english.

Author: I'm keeping the term aggregated because that's how we refer to this type of calculation in the docs, but I've adjusted the rest of the text.

> For spending-limit computation, each payment will be timestamped at the first second of the first minute of the hour, and each hour will reset the rolling period.

Reviewer: Do you mean:
```suggestion
When we calculate spending limits, each payment will be timestamped at the first second of the first minute of the hour. The rolling period resets every hour.
```
?

The rest is technical lingo, it's hard to be more abstract for a developer audience :)
